cpp/src/lance/format/schema.h

```diff
@@ -14,6 +14,7 @@
 
 #pragma once
 
+#include <arrow/compute/exec/expression.h>
```

Comment:
Is this Arrow public API? Or is there a higher level header file for this?

Reply:
There is a `arrow/compute/api.h` should i use that one?